src/Analysis/GWASV2/Components/Covariates/CovariatesCardList.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { DeleteOutlined } from '@ant-design/icons';
+import CovariatesCardsList from './CovariatesCardsList';
+import ValidInitialState from '../../TestData/InitialStates/ValidInitialState';
+
+describe('CovariatesCardsList component', () => {
+  let wrapper;
+  const mockDeleteCovariate = jest.fn();
+
+  beforeEach(() => {
+    const mockProps = {
+      outcome: ValidInitialState.outcome,
+      covariates: ValidInitialState.covariates,
+      deleteCovariate: mockDeleteCovariate,
+    };
+    wrapper = mount(<CovariatesCardsList {...mockProps} />);
+  });
+
+  it('should render an outcome card', () => {
+    expect(wrapper.find('.outcome-card').exists()).toBe(true);
+    expect(wrapper.find('.outcome-card .ant-card-meta-title').text()).toBe(
+      'Outcome Phenotype'
+    );
+    expect(
+      wrapper.find('.outcome-card .ant-card-meta-description').text()
+    ).toBe('Attribute8');
+  });
+
+  it('should render covariate cards', () => {
+    expect(wrapper.find('.dichotomous-card').exists()).toBe(true);
+    expect(
+      wrapper
+        .find('.dichotomous-card .ant-card-meta-title')
+        .last()
+        .text()
+    ).toBe('Dichotomous Covariate');
+    expect(
+      wrapper
+        .find('.dichotomous-card .ant-card-meta-description')
+        .last()
+        .text()
+    ).toBe('test dichotomous covariate');
+
+    expect(wrapper.find('.continuous-card').exists()).toBe(true);
+    expect(
+      wrapper
+        .find('.continuous-card .ant-card-meta-title')
+        .last()
+        .text()
+    ).toBe('Continuous Covariate');
+    expect(
+      wrapper
+        .find('.continuous-card .ant-card-meta-description')
+        .last()
+        .text()
+    ).toBe('Attribute83');
```

Comment:
Code clarity comment: it is not immediately obvious why this is a correct expectation. Only after understanding that it is in `ValidInitialState`, it becomes clear. It also feels like `ValidInitialState` is being misused a bit as it is not an initial state anymore, but rather more like a "filled state" that is useful for testing different (not all) scenarios. Maybe calling it something like "MockTestState" instead of `ValidInitialState` makes it a bit clearer.

Reply:
This has been updated: the strings use variable names that get the value from the ValidState. The naming InitialStates/ValidInitialState.js has been changed to States/ValidState.js